src/interfaces/filing-interfaces/filing-data-interface.ts

```diff
@@ -1,5 +1,5 @@
 import { FilingCodes } from '@/enums'
-import { CorpTypeCd } from '@bcrs-shared-components/enums/'
+import { CorpTypeCd } from '@bcrs-shared-components/corp-type-module/'
```

Comment:
This is to avoid this "error":

![image](https://github.com/bcgov/business-create-ui/assets/10002889/340f05eb-9c0d-4447-a84b-e6055ade806e)

For some reason, when CorpTypeCd is re-exported (in index.ts), the linter/language module doesn't recognize it to be the same enum and reports this error. Importing CorpTypeCd directly from the module works around this issue.

I consider this to be a language bug and I hope it will be resolved soon, but in the meantime this is an innocuous work-around.